### DISCUSSION TITLE
Building documentation locally

### DISCUSSION THREAD

**Discussion by jyoo980** [syl20bnr/spacemacs#16341] [Q&A] [answered]
Hello,

I have some changes to correct the formatting of some of the documentation in the shell layer. I have edited `layers/+tools/shell/README.org` and would like to validate my changes locally before opening a PR.

I am not able to find any documentation on how I can locally build or deploy the documentation website, are these steps documented somewhere?

**Comment by smile13241324** [comment, marked as answer]
This is documented as part of the CI documentation, see here https://github.com/syl20bnr/spacemacs/blob/develop/doc/CI_PLUMBING.org#documentation-updates.

**Comment by smile13241324** [comment]
Question has been answered.
